rudder-reports/debian/postinst

```diff
@@ -19,62 +19,69 @@ set -e
 
 
 case "$1" in
-    configure)
-		invoke-rc.d rsyslog restart
```

Comment:
Because it has never been useful in the first place (not commented , and actually comes from the fact that this postinst was copied from another package)